GraphicsProgram/strategies/Circle/CircleIfUserOperation.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Drawing;
+using GraphicsProgram;
+using GraphicsProgram.Shapes;
+
+public class CircleIfUserOperation : IUserOperationStrategy
+{
+    public bool AppliesTo(string userOperationType, string shape)
+    {
+        return userOperationType.Equals(OperationType.If) && shape.Equals(ShapeType.Circle);
+    }
+
+    public void DoDrawing(Pen pen, PenPosition penPosition, Graphics g, string line)
```

Comment:
Is currently where a circle is created based on the input 'circle 100'?

Reply:
Yeah 😄 the paramter `line` contains the full string, which is then split on line 15.